test/integration/amp-handler-test.js

```diff
@@ -104,9 +104,21 @@ function createApp(app = express()) {
   return app;
 }
 
+function createStoryPageApp(app = express()) {
+  const ampLibrary = {};
+  ampLibrary.ampifyStory = () =>
+    '<div data-page-type="story-page">foobar</div>';
+  ampRoutes(app, {
+    getClient: getClientStub,
+    publisherConfig: {},
+    ampLibrary,
+  });
+  return app;
+}
+
 describe("AmpHandler", () => {
   it("mounts an amp page", (done) => {
-    const app = createApp();
+    const app = createHomePageApp();
```

Comment:
Don't need two apps, you can test both routes on the same app

Reply:
okay.